libs/ngx-charts-on-fhir/src/lib/timeline-range-selector/timeline-range-selector.component.ts

```diff
@@ -40,6 +41,19 @@ export class TimelineRangeSelectorComponent {
       this.selectedButton = this.findMatchingButton(this.selectedDateRange);
       this.changeDetectorRef.markForCheck();
     });
+    window.addEventListener('keydown', this.handleKeyboardZoom.bind(this));
+  }
+
+  handleKeyboardZoom(event: KeyboardEvent): void {
+    if (ZOOM_KEYS.includes(event.key)) {
+      const zoomFactor = event.key === '+' ? 0.9 : 1.1;
+      const currentRange = this.selectedDateRange;
+      if (currentRange.start && currentRange.end) {
+        const newMin = currentRange.start.getTime() + (currentRange.end.getTime() - currentRange.start.getTime()) * (1 - zoomFactor);
+        const newMax = currentRange.end.getTime() - (currentRange.end.getTime() - currentRange.start.getTime()) * (1 - zoomFactor);
+        this.configService.zoom({ min: newMin, max: newMax });
+      }
+    }
```

Comment:
why are we handling zoom and pan in different components? can we support keyboard zoom when timeline-range-selector is not used? would it help to add `zoomIn()` and `zoomOut()` functions to configService?